cmd/historyAccounts.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// historyAccountsCmd represents the historyAccounts command
+var historyAccountsCmd = &cobra.Command{
+	Use:   "historyAccounts",
```

Comment:
Nit: for naming consistency with existing Blockchain ETL projects, let's snake-case the `cobra.Command.Use` fields and prefix the table name with `export_`, like so: `export_history_accounts`.